packaging/rpm/microshift.spec

```diff
@@ -509,6 +568,18 @@ fi
 %files multus-release-info
 %{_datadir}/microshift/release/release-multus-{x86_64,aarch64}.json
 
+%files flannel
+%dir %{_prefix}/lib/microshift/manifests.d/000-microshift-flannel
+%dir %{_prefix}/lib/microshift/manifests.d/000-microshift-kube-proxy
+%{_prefix}/lib/microshift/manifests.d/000-microshift-flannel/*
+%{_prefix}/lib/microshift/manifests.d/000-microshift-kube-proxy/*
+%config(noreplace) %{_sysconfdir}/microshift/config.d/disableDefaultCNI.yaml
```

Comment:
Fix is needed here to use `00-...` name.

Reply:
Done